package.json

```
@@ -79,12 +79,14 @@
   "dependencies": {
     "@babel/runtime": "^7.1.2",
     "@semantic-ui-react/event-stack": "^3.1.0",
+    "@stardust-ui/react-component-event-listener": "^0.37.0",
```

```suggestion
    "@stardust-ui/react-component-event-listener": "~0.37.0",
```

Use more accurate versions